libbackscrub.cc

```diff
@@ -0,0 +1,272 @@
+/* This is licenced software, @see LICENSE file.
+ * Authors - @see AUTHORS file.
+==============================================================================*/
+
+// tested against tensorflow lite v2.4.1 (static library)
+
+#include "tensorflow/lite/interpreter.h"
+#include "tensorflow/lite/kernels/register.h"
+#include "tensorflow/lite/model.h"
+#include "tensorflow/lite/optional_debug_tools.h"
+
+#include "transpose_conv_bias.h"
+#include "libbackscrub.h"
+
+// Debug helper
+static void _dbg(calcinfo_t &info, const char *fmt, ...) {
+	va_list ap;
+	va_start(ap, fmt);
+	if (info.ondebug)
+		info.ondebug(info.caller_ctx, fmt, ap);
+	else
+		vfprintf(stderr, fmt, ap);
+	va_end(ap);
+}
+
+using namespace tflite;
```

Comment:
Absolutely not a fan. Especially on a global source level.

Reply:
Yeah, I don't like `using` either - nuking.